hc/api/tests/test_create_check.py

```diff
@@ -77,5 +79,35 @@ def test_it_rejects_non_string_name(self):
         self.post({"api_key": "abc", "name": False},
                   expected_error="name is not a string")
 
-    ### Test for the assignment of channels
-    ### Test for the 'timeout is too small' and 'timeout is too large' errors
+    def test_timeout_is_too_small(self):
+        r = self.post({"api_key": "abc", "timeout": 1},
+                      expected_error="timeout is too small")
+
+        j = r.json()
+        self.assertEqual(j["error"], "timeout is too small")
+
+    def test_timeout_is_too_large(self):
+        r = self.post({"api_key": "abc", "timeout": 3456567},
+                      expected_error="timeout is too large")
+
+        j = r.json()
+        self.assertEqual(j["error"], "timeout is too large")
+
+    def test_for_the_assignment_of_channels(self):
+
+        channel = Channel(user=self.alice, kind="pushbullet")
+        channel.save()
+
+        channel2 = Channel(user=self.alice, kind="pushbullet")
+        channel2.save()
+
+        data = {"api_key": "abc", "name": "Kip", "tags": "sims,cps", "timeout": 3600,
+                "grace": 60, "channels": "*"}
+
+        r = self.client.post(self.URL, json.dumps(
+            data), content_type="application/json")
+
+        channels = Channel.objects.all()
+
+        for channel in channels:
+            self.assertNotEqual(channel.check, None)
```

Comment:
Still not complete. 
You should be checking the `Check` object. Refer to how `assign_all_channels` works.